src/bin/CMakeLists.txt

```diff
@@ -269,7 +269,7 @@ endif()
 file(GLOB PUBLIC_HEADERS "../inc/*.h" "../inc/*.hpp")
 
 if(BUILD_SHARED_LIBS)
-  install(TARGETS msquic EXPORT msquic DESTINATION lib)
+  install(TARGETS msquic platform inc logging_inc warnings main_binary_link_args OpenSSL OpenSSLQuic EXPORT msquic DESTINATION lib)
```

Comment:
OpenSSL isn't always used.

Reply:
I know. I write that on the description on this PR.
Can you suggest me how to do it more general?

Reply:
There is other targets in other tls options?

Reply:
I think I solved it.